packages/adaptive-ui/src/index.ts

```diff
@@ -2,6 +2,7 @@ export * from "./color/index.js";
 export * from "./density/index.js";
 export * from "./elevation/index.js";
 export * from "./modules/index.js";
+export * from "./design-tokens/modules.js";
```

Comment:
The location of `modules.ts` is intentional as it also represents an opinion on a reference implementation of Adaptive UI.

I've been thinking of the split where the root export contains all of the tools that enable adaptive styling, including the recipes, helper functions, and style generation.

The `reference` export contains all the opinions about styling including named tokens and groups. Adaptive Web Components is currently an implementation of components on the reference opinions. The ideal state is that some day the reference implementation can be described in some format (json tokens) and generated at build or run time.

Let's revert this and the change in `design-tokens/index.ts`.